### ISSUE TITLE
Why in version 223 - CC API VERSION 2.42.0, the method changed the Response (http code & body result)?

### ISSUE THREAD

**Issue by jabrena** [cloudfoundry/cloud_controller_ng#482]
Good morning,

I would like to know why this method changed the REST output contract in relation to previous version.
http://apidocs.cloudfoundry.org/223/apps/remove_service_binding_from_the_app.html

From 2.44 to last version, this method returns a HTTP CODE 204 without BODY content but in previous versions, the method returned a 201 and a JSON.

What is the reason to change the contract?
Is it a bug?

I have tested this fact in 3 environments:

``` bash
Environment: BLUEMIX 2.40.0 <- 201 JSON
Environment: PIVOTAL 2.44.0 <- 204 NO BODY
Environment: LOCAL_INSTANCE_1 2.25.0 <- 201 JSON
```

Many thanks in advance.

Juan Antonio

**Comment by dieucao** [comment]
@jabrena This was posted to cf-dev [1] a while back asking if anyone had any issues with CC changing the response code from 201 Created is incorrect for DELETE end points.

Is this causing an issue for you?

[1] https://lists.cloudfoundry.org/archives/list/cf-dev@lists.cloudfoundry.org/thread/6X2HPOUB7O3ICY4DDMS27QTW7V7WGM7V/#6X2HPOUB7O3ICY4DDMS27QTW7V7WGM7V

**Comment by jabrena** [comment]
Good morning @dieucao,

yes, I was checking the whole project and I noticed that every method which it removes an objects returns 204 so it is a good change.

If I detect some similar doubts, I will notify.

At the moment, I have registered 3 http codes:
https://github.com/prosociallearnEU/cf-nodejs-client/blob/master/lib/utils/HttpCodes.js

The alternative for this method:
http://apidocs.cloudfoundry.org/226/service_bindings/delete_a_particular_service_binding.html

has the right http code.

Cheers
